### PR TITLE
refs #4917 - add realm ad plugin support

### PR DESCRIPTION
**Pull request by timogoebel**

### CODE REVIEW

**Comment by timogoebel** [comment]
Test failures look unrelated.

File: templates/realm.yml.erb [hunk 1]
```diff
@@ -4,6 +4,7 @@
 :enabled: <%= @module_enabled %>
 
 # Available providers:
+#   ad
```
Comment (ekohl) [inline templates/realm.yml.erb]: Is this ad or ```realm_ad```?

Comment (timogoebel) [reply templates/realm.yml.erb]: The latter, of course. :-)

**Comment by timogoebel** [comment]
@ekohl: Fixed the comment. Packaging is done. Anything more to do here?

**Comment by ekohl** [comment]
Restarted a test that failed due to timeouts.

**Comment by timogoebel** [comment]
All 💚  . :-)

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**